### PR TITLE
feat: distinguish between cache and connection pool region

### LINKED ISSUE #5237: Distinguish between locations of caching nodes and regions of the connection pool.
Document publicly in which regions Accelerate’s connection pool is available

### PR DESCRIPTION
**Pull request by ankur-arch**
closes #5237

### CODE REVIEW

File: content/800-data-platform/100-accelerate/600-faq.mdx [hunk 1]
```diff
@@ -33,11 +33,15 @@ This might be an issue with the TypeScript server not yet recognizing the regene
...
 VS Code should now recognize the `$extends` method.
 
-## What regions are Accelerate available in?
+## What regions are Accelerate's cache nodes available in?
 
-Accelerate runs on Cloudflare's network. Cache hits are served from Cloudflare's 300+ locations. Cache misses can be served from 16 different locations at the time of writing. More will be added in the future.
+Accelerate runs on Cloudflare's network and cache hits are served from Cloudflare's 300+ locations. You can find the regions where Accelerate's cache nodes are available here: [https://www.cloudflare.com/network/](https://www.cloudflare.com/network/).
 
-You can find the regions Accelerate is available here: [https://www.cloudflare.com/network/](https://www.cloudflare.com/network/).
+## What regions are Accelerate's connection pool available in?
```
Comment (nikolasburk) [inline content/800-data-platform/100-accelerate/600-faq.mdx]:
```suggestion
## What regions is Accelerate's connection pool available in?
```

File: content/800-data-platform/100-accelerate/600-faq.mdx [hunk 2]
```diff
@@ -33,11 +33,15 @@ This might be an issue with the TypeScript server not yet recognizing the regene
...
+
+When no cache strategy is specified or in the event of a cache miss, the request is routed through Accelerate's connection pool. Currently, requests can be routed through any chosen region among the 16 available locations.
```
Comment (nikolasburk) [inline content/800-data-platform/100-accelerate/600-faq.mdx]:
```suggestion
When no cache strategy is specified or in the event of a cache miss, the Prisma Client query is routed through Accelerate's connection pool. Currently, queries can be routed through any chosen region among the 16 available locations.
```

**Review by nikolasburk — CHANGES_REQUESTED** [review]
This looks great, thanks for picking it up so quickly Ankur!

FWIW, I would prefer to list the available regions here even that means they need to be hardcoded (for now).

I think the value of including them here so that people who are _evaluating_ Accelerate and who might not have a PDP account yet (thus not being able to follow the link) can still find them is more important and should be prioritized over this list “getting out of sync with its source of truth”.

File: content/800-data-platform/100-accelerate/600-faq.mdx [hunk 3]
```diff
@@ -33,11 +33,34 @@ This might be an issue with the TypeScript server not yet recognizing the regene
...
-Accelerate runs on Cloudflare's network. Cache hits are served from Cloudflare's 300+ locations. Cache misses can be served from 16 different locations at the time of writing. More will be added in the future.
-
-You can find the regions Accelerate is available here: [https://www.cloudflare.com/network/](https://www.cloudflare.com/network/).
+## What regions are Accelerate's cache nodes available in?
+
+Accelerate runs on Cloudflare's network and cache hits are served from Cloudflare's 300+ locations. You can find the regions where Accelerate's cache nodes are available here: [https://www.cloudflare.com/network/](https://www.cloudflare.com/network/).
+
+## What regions is Accelerate's connection pool available in?
+
+When no cache strategy is specified or in the event of a cache miss, the Prisma Client query is routed through Accelerate's connection pool. Currently, queries can be routed through any chosen region among the 16 available locations.
```
Comment (janpio) [inline content/800-data-platform/100-accelerate/600-faq.mdx:42]: Also when the Accelerate extension is not used (as it is not needed for pure connection pooling needs)

**Review by janpio — APPROVED** [review]

File: content/800-data-platform/100-accelerate/600-faq.mdx [hunk 4]
```diff
@@ -33,11 +33,34 @@ This might be an issue with the TypeScript server not yet recognizing the regene
...
+- Europe (Frankfurt)
+- Europe (Ireland)
+- Europe (London)
+- Europe (Paris)
+- Europe (Stockholm)
+- South America (Sao Paolo)
+- US East (N. Virginia)
+- US East (Ohio)
+- US West (N. California)
+- US West (Oregon)
```
Comment (nikolasburk) [inline content/800-data-platform/100-accelerate/600-faq.mdx]: don't all of these have region code, like `eu-west-1` or something like this? if so, could we add these codes here?

Comment (ankur-arch) [reply content/800-data-platform/100-accelerate/600-faq.mdx]: I copy pasted the user friendly region names. 
@nikolasburk With the region codes, how should the format be?

Comment (nikolasburk) [reply content/800-data-platform/100-accelerate/600-faq.mdx]: I would suggest smth like:

- `eu-west-2` (Mumbai | Asia Pacific)

or

- `eu-west-2` (Mumbai, Asia Pacific)

Comment (nikolasburk) [reply content/800-data-platform/100-accelerate/600-faq.mdx]: or some other combination of code formatted region, parantheses, commas and pipes 😅

Comment (ankur-arch) [reply content/800-data-platform/100-accelerate/600-faq.mdx]:
```suggestion
- Asia Pacific (Mumbai) - `ap-south-1`
- Asia Pacific (Seoul) - `ap-northeast-2`
- Asia Pacific (Singapore) - `ap-southeast-1`
- Asia Pacific (Sydney) - `ap-southeast-2`
- Asia Pacific (Tokyo) - `ap-northeast-1`
- Canada (Central) - `ca-central-1`
- Europe (Frankfurt) - `eu-central-1`
- Europe (Ireland) - `eu-west-1`
- Europe (London) - `eu-west-2`
- Europe (Paris) - `eu-west-3`
- Europe (Stockholm) - `eu-north-1`
- South America (Sao Paulo) - `sa-east-1`
- US East (N. Virginia) - `us-east-1`
- US East (Ohio) - `us-east-2`
- US West (N. California) - `us-west-1`
- US West (Oregon) - `us-west-2`
```

File: content/800-data-platform/100-accelerate/600-faq.mdx [hunk 5]
```diff
@@ -33,11 +33,34 @@ This might be an issue with the TypeScript server not yet recognizing the regene
...
+- Europe (Frankfurt) - `eu-central-1`
+- Europe (Ireland) - `eu-west-1`
+- Europe (London) - `eu-west-2`
+- Europe (Paris) - `eu-west-3`
+- Europe (Stockholm) - `eu-north-1`
+- South America (Sao Paulo) - `sa-east-1`
+- US East (N. Virginia) - `us-east-1`
+- US East (Ohio) - `us-east-2`
+- US West (N. California) - `us-west-1`
+- US West (Oregon) - `us-west-2`
```
Comment (ankur-arch) [inline content/800-data-platform/100-accelerate/600-faq.mdx]:
```suggestion
- Asia Pacific (Mumbai) `ap-south-1`
- Asia Pacific (Seoul) `ap-northeast-2`
- Asia Pacific (Singapore)  `ap-southeast-1`
- Asia Pacific (Sydney) `ap-southeast-2`
- Asia Pacific (Tokyo) `ap-northeast-1`
- Canada (Central) `ca-central-1`
- Europe (Frankfurt) `eu-central-1`
- Europe (Ireland) `eu-west-1`
- Europe (London) `eu-west-2`
- Europe (Paris) `eu-west-3`
- Europe (Stockholm) `eu-north-1`
- South America (Sao Paulo) `sa-east-1`
- US East (N. Virginia) `us-east-1`
- US East (Ohio) `us-east-2`
- US West (N. California) `us-west-1`
- US West (Oregon) `us-west-2`
```

Comment (nikolasburk) [reply content/800-data-platform/100-accelerate/600-faq.mdx]: how about this format: 

- US West, Oregon (`us-west-2`)

Comment (ankur-arch) [inline content/800-data-platform/100-accelerate/600-faq.mdx]:
```suggestion
- Asia Pacific, Mumbai (`ap-south-1`)
- Asia Pacific, Seoul (`ap-northeast-2`)
- Asia Pacific, Singapore (`ap-southeast-1`)
- Asia Pacific, Sydney (`ap-southeast-2`)
- Asia Pacific, Tokyo (`ap-northeast-1`)
- Canada, Central (`ca-central-1`)
- Europe, Frankfurt (`eu-central-1`)
- Europe, Ireland (`eu-west-1`)
- Europe, London (`eu-west-2`)
- Europe, Paris (`eu-west-3`)
- Europe, Stockholm (`eu-north-1`)
- South America, Sao Paulo (`sa-east-1`)
- US East, N. Virginia (`us-east-1`)
- US East, Ohio (`us-east-2`)
- US West, N. California (`us-west-1`)
- US West, Oregon (`us-west-2`)
```

**Review by nikolasburk — APPROVED** [review]

**Pull request merged**